staging/kos/cmd/attachment-tput-driver/main.go

```diff
@@ -102,6 +102,18 @@ var (
 	failedDeletes              prometheus.Counter
 )
 
+// The following vars are used by threads to compute the time at which the next
+// op should take place.
+var (
+	// Each thread must perform at least `opsPerThdMin` ops on attachments
+	// (but some threads will perform more).
+	opsPerThdMin uint64
+	// `attsLeft` is the number of attachments that would be "in excess" if we
+	// wanted all threads to manage (create and delete) the same number of
+	// attachments.
+	attsLeft uint64
+)
+
```

Comment:
I would describe it thusly.  The problem is that when the number of attachments is not a multiple of the number of threads some threads create and delete one more attachment than others and thus need to advance through the schedule with a shorter stride.  What's called `opsPerThdMin` here is the number of regular strides taken through the schedule, and what's called here `attsLeft` is the shorter stride that some threads use at the end.